entities.py

```diff
@@ -96,10 +133,33 @@ def wrapper(self, *args, **kwargs):
 
     # Actions
     def attack(self, other):
+        """
+        Attack another entity using a variety of possible attacks.
+
+        Args:
+            other (Entity): Entity to attack.
+        """
         # TODO: Work out differents types of damage
         other.hp -= self.physical_dmg
 
     def take_turn(self, target, game_map):
+        """
+        Let the current actor take a turn.
+
+        A turn is defined by the usual actions that an Actor can perform
+        before other Actos can take a turn.
+
+        Examples:
+            * Moving
+            * Attacking
+            * Consuming / using items
+            * Activating mechanisms
+            * Picking up items
+
```

Comment:
You could just say that the turn is defined by its behavior, since it delegates the logic to the corresponding Behavior subclass, and just put this in the Behavior docstring.